src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/DataSetDetailsViewModel.cs

```diff
@@ -0,0 +1,46 @@
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
+
+public record DataSetDetailsViewModel
+{
+    public string Title { get; set; } = null!;
+    public string Summary { get; set; } = null!;
+    public DataSetDetailsFileViewModel File { get; set; } = null!;
+    public DataSetDetailsReleaseViewModel Release { get; set; } = null!;
```

Comment:
Just wondering if we should have blank lines separating fields to be consistent with the spacing in other models like `DataSetDetailsReleaseViewModel` where annotations force the blank lines.

I've been wondering this in some of my own changes recently. Hopefully the code standards coming in soon will enforce this one way or the other.

Reply:
I put blank lines on the one view model because it had the `JsonConvertor` on one property - the rest I figured I could condense. I've added spaces between the properties of the others as well now.

I'd not be inclined to stipulate a way in coding standards, to be honest - I don't think it matters.

Reply:
Looks good now. 👍